.github/workflows/docker-publish-latest.yaml

```diff
@@ -5,7 +5,7 @@ on:
       - main
     paths:
       - ".github/workflows/docker-publish-latest.yaml"
-      - "backend/**"
+      - "Dockerfile"
```

Comment:
This line indicates that the `docker-publish-latest` job will run whenever the `Dockerfile` is changed. We also need to trigger the workflow when the backend source code is updated. Therefore, the workflow should run if there are changes in either the `backend/**` directory or the Dockerfile.

Reply:
I got it. Thank you !